tasks/component-builder.js

```diff
@@ -345,3 +359,9 @@ async function main({
 exports.processCSS = processCSS;
 exports.fetchContent = fetchContent;
 exports.default = main;
+
+let {
+	_: components,
+} = yargs(hideBin(process.argv)).argv;
+
+Promise.all(components.map((componentName) => main({ componentName })));
```

Comment:
I'm curious about what this is doing with yargs and the change to run main() on all components at the end of this script. And the same question for the similar change in `tasks/component-reporter.js`.

Reply:
It's a way to help support running these scripts as node entry points vs. calling the functions in the modules. A difference between saying: `node ./tasks/component-builder.js' or having to extract the default method and run it: `node -e 'import('./tasks/component-builder.js').default()`